src_c/key.c

```diff
@@ -804,13 +804,35 @@ key_set_text_input_rect(PyObject *self, PyObject *obj)
     /* https://wiki.libsdl.org/SDL_SetTextInputRect */
 #if IS_SDLv2
     SDL_Rect *rect, temp;
+    SDL_Window *sdlWindow = pg_GetDefaultWindow();
+    SDL_Renderer *sdlRenderer = SDL_GetRenderer(sdlWindow);
+
     if (obj == Py_None) {
         Py_RETURN_NONE;
     }
     rect = pgRect_FromObject(obj, &temp);
     if (!rect)
         return RAISE(PyExc_TypeError, "Invalid rect argument");
+
+    if (sdlRenderer != NULL) {
+        SDL_Rect vprect, rect2;
```

Comment:
I'm surprised this doesn't crash the python2 compiler...

Reply:
It's at the start of a block, that should be enough.